### ISSUE TITLE
Clicking on links in chat doesn't work

### ISSUE THREAD

**Issue by Un1q32** [Wilyicaro/Legacy-Minecraft#418]
http and https links in chat are not clickable like they are in vanilla

**Comment by Wilyicaro** [comment]
No, click events are working normally as in vanilla

**Comment by Un1q32** [comment]
nope, modded click events work, but if a player sends "https://google.com" in chat you cant click the link

**Comment by Wilyicaro** [comment]
You can't do this in vanilla either, "modded" click events work the same way as link click events

**Comment by Un1q32** [comment]
huh, must have been a mod i used before, I could have sworn you could do that in vanilla
